doc/snek.adoc

```diff
@@ -1225,9 +1224,13 @@ as indicating _base_ and _bound_. Here are some examples:
 === String Interpolation
 
 String interpolation in Snek can be confused with formatted printing
-in other languages. In Snek, the ``print`` function takes a single
-String. String interpolation is how this String is generated from a format
-specification String and a List or Tuple of parameters.(((%)))(((string interpolation)))
+in other languages. In Snek, the ``print`` function takes String
```

Comment:
actually, like python3, print takes *any* values. I fixed the 'print' reference, but missed this one.

Reply:
Fixed.